src/side-navigation/__tests__/side-navigation.test.tsx

```diff
@@ -3,6 +3,8 @@
 import * as React from 'react';
 import { render } from '@testing-library/react';
 
+import Multiselect from '../../../lib/components/multiselect';
+import Select from '../../../lib/components/select';
```

Comment:
The new property is just a slot - there is no need to test if it can render different components. The test can use a fake component or maybe just select so that we can ensure the `wrapper.findItemsHeader()!.findSelect()` chaining works.